### ISSUE TITLE
running jirecon.sh Error: Could not find or load main class org.jitsi.jirecon.Main

### ISSUE THREAD

**Issue by ghost** [jitsi/jirecon#21]
After I run the jirecon.sh  it gives me this:

./jirecon.sh --conf=jirecon.properties --time=10 someID@xxx.server.com

Error: Could not find or load main class org.jitsi.jirecon.Main

**Comment by khatanbaatar** [comment]
first. jirecon directory `ant jar`` command run
second. ./jirecon.sh --conf = jirecon.properties --time = 10 someID@xxx.server.com

**Comment by khatanbaatar** [comment]
but this project is not working.

**Comment by khatanbaatar** [comment]
https://github.com/ubity/jirecon this work. testing

**Comment by dakshhmehta** [comment]
The ant command before the actual command did the trick.

However, somehow the jirecon user is not able to join.

**Comment by shierro** [comment]
Yes same case with me. Jirecon user is not able to join. were you able to connect jirecon to XMPP server @khatanbaatar ?
